### PR TITLE
Include less v16

### PR DESCRIPTION
**Pull request by catenacyber**
Link to [redmine](https://redmine.openinfosecfoundation.org/projects/suricata/issues) ticket:
https://redmine.openinfosecfoundation.org/issues/945

Describe changes:
- Remove unused includes in c files
- adds some options in CI : hyperscan, luajit

Follows #8142 with 
- completing removal of double inclusion, like src/runmode-netmap.c was now including twice "util-time.h"
- Rebase (especially after app-layer-dcerpc-common.h was removed by another PR)

Still TODO after that : run script mininc.py

### CODE REVIEW

**Comment by suricata-qa** [comment]
Information: QA ran without warnings. 

Pipeline 10609

**Review by victorjulien — CHANGES_REQUESTED** [review]
minor changes requested, ready to go after that I think

File: src/alert-syslog.c [hunk 1]
```diff
@@ -48,6 +48,8 @@
 #include "util-syslog.h"
 #include "util-optimize.h"
 #include "util-logopenfile.h"
+// ACTION_DROP
```
[the remark this answers is not in the document]
Comment (catenacyber) [reply src/alert-syslog.c:51]: Ok

**Comment by catenacyber** [comment]
Replaced by https://github.com/OISF/suricata/pull/8174

**Pull request closed without merging**